dcos/dcos.go

```diff
@@ -4,17 +4,19 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"os"
+	"strings"
 
 	"github.com/libopenstorage/secrets"
 	api "github.com/portworx/dcos-secrets"
 )
 
 // Keys for the config to initialize the DC/OS secrets client
 const (
-	KeyUsername   = "username"
-	KeyPassword   = "password"
-	KeyCACertFile = "cacert"
-	KeyDcosURL    = "dcos_url"
+	EnvSecretsUsername   = "DCOS_SECRETS_USERNAME"
```

Comment:
Will this require a configuration change for existing customers?

Reply:
Currently this is only used by Portworx, where it reads these environment variables and puts them in the map under `username` and `password` keys. Going to change Portworx code to send these env vars as keys directly in the map or not send it at all. In that case we will read those from env vars in this package.